system/core.go

```diff
@@ -91,7 +100,7 @@ func (application *Application) Close() {
 	glog.Info("Bye!")
 }
 
-func (application *Application) Route(controller interface{}, route string) interface{} {
```

Comment:
Despite this one improvement (as I see it) here, I'm not fond of how `Route` works, particularly it's taking `controller interface{}` and using reflect to call a method.  Perhaps this was to break the `system` <--> `controllers` dependency cycle?  I'd much to prefer to pass a function with a certain signature, as in the [newly added APIHandler](https://github.com/decred/dcrstakepool/pull/73/files#diff-5467de60ded0da2fb4a582d6bfa29bddR148).  But that's for another day.